markdown.js

```diff
@@ -0,0 +1,103 @@
+const visit = require('unist-util-visit');
+var remark = require('remark');
+var parse = require('remark-parse');
+var html = require('remark-html');
+var fs = require('fs');
+
+var file = fs.readFileSync('./src/content/plugins/index.md', { encoding: 'utf8' });
+
+remark()
+  .use(parse)
+  .use(md)
+  .use(html)
+  .process(file, function(err, file) {
+    if (file) {
+      console.log(file.contents);
```

Comment:
this is weird, are we logging this for progress? maybe add some text

Reply:
Will be removed, since this file is used as webpack plugin, it is not efficient to restart your webpack compilation to test it, so, I used the plugin as a standalone node program, faster to kill and restart.